### ISSUE TITLE
`ExpandablePanel` should not use `height` on the host after animation finish

### ISSUE THREAD

**Issue by jarrodek** [elix/elix#124]
The `ExpandablePanel` assumes that children added to the panel won''t change (at least I can't see any code related to it). When a content is rendered (`opened` property is set to `true`) and after removing an item from the children, the panel keeps the original height of the content.

I am using the `ExpandablePanel` in an navigation element. It renders a list of items. The navigation items, however, changes for given conditions and and if the tree leaf was opened and a child was removed it renders a gap. I need to close and open the panel to mitigate this.

I think that after finishing the animation the panel should remove the `height` from the container's styles.

**Comment by JanMiksovsky** [comment]
Hmm, interesting.

Can you verify that you're using Chrome/Edge? I can't repro this in Safari or Firefox.

In Chrome, if I use dev tools to append content to an opened ExpandablePanel, the content does not appear. But if I interact with dev tools, the content refreshes. I think some part of dev tools must be forcing a recalc of the height.

In which case, I think this might be a Chromium bug. In any event, the ExpandablePanel only sets height during the transition, then it clears it after the effect has completed. So I'm not sure what more we can do beyond forcing a height recalc ourselves.

**Comment by jarrodek** [comment]
Interesting. So, this happened on Chrome. In a `LitELement` controlled element I was adding and removing nodes in a template. For a long time I thought I have some bug in my program but it turned out that the ExpandablePanel did not refresh the view on change and new items simply weren't rendered. In dev tools I checked that the nodes exists outside the view and the content of the `ExpandablePanel` has height set. Then I reported the issue.
